packages/aragon-wrapper/src/index.js

```diff
@@ -231,6 +231,7 @@ export default class Aragon {
     this.initAppIdentifiers()
     this.initNetwork()
     this.initNotifications()
+    this.trigger = new Subject()
```

Comment:
Rather than having a global "trigger" message bus, I've been thinking more and more about having application-specific contexts to manage this better (ie. anything that's not really relevant for a client to subscribe to).

A few things fit:

- App triggers
- App state (the localstorage / indexeddb caching will be turned into a side effect of the state soon)
- Perhaps app paths

And maybe more.

Reply:
An initial version of this context has been implemented in https://github.com/aragon/aragon.js/pull/352 ([implementation](https://github.com/aragon/aragon.js/blob/7e8f263d8610fdb038427535b8eb9d3f1e8747ae/packages/aragon-wrapper/src/apps/index.js)).